private/buf/bufgen/generator.go

```diff
@@ -295,6 +295,12 @@ func (g *generator) execLocalPlugin(
 	if err != nil {
 		return nil, err
 	}
+	options := []appprotoexec.GenerateOption{
+		appprotoexec.GenerateWithPluginPath(pluginConfig.Path...),
+	}
+	if pluginConfig.ProtocPath != "" {
+		options = append(options, appprotoexec.GenerateWithProtocPath(pluginConfig.ProtocPath))
```

Comment:
You don't need to protect this with an if, as if it is empty, it is as if the option is not called